components/cam/src/physics/cam/check_energy.F90

```diff
@@ -590,7 +590,9 @@ subroutine check_energy_gmean(state, pbuf2d, dtime, nstep)
 !-----------------------------------------------------------------------
 
     ! Copy total energy out of input and output states
+#ifdef CPRCRAY
 !DIR$ CONCURRENT
+#endif
```

Comment:
Cray compiler directive that may be needed when porting and optimizing to Cray. `CPRCRAY` macro cleans up Intel compiler messages like
```
/qfs/people/mame117/repos/E3SM-tests/components/cam/src/physics/cam/check_energy.F90(593): remark #5140: Unrecognized directive
!DIR$ CONCURRENT
----------------^
```